soap/soap_connector.bal

```diff
@@ -48,3 +65,23 @@ public function SoapConnector::sendSoapRequest(string path, SoapRequest request)
         }
     }
 }
+
+public function SoapConnector::sendRobust(string path, SoapRequest request) returns (()|SoapError) {
```

Comment:
You can use `returns error?`

Reply:
Then I have to return an `error` definitely. But in the implementation of this method there is a scenario where `nill` should be returned. So, how can I resolve that?

Reply:
`error?` means error or nil. It is a short form.

Reply:
Got it and fixed. Thanks.